### DISCUSSION TITLE
How to find CNCF contributors from a specific country?

### DISCUSSION THREAD

**Discussion by aliok** [cncf/devstats#29] [Q&A] [answered]
We have a mentoring program in Turkey, where the mentors work with mentees help them make their first open source contributions.

What would be super nice is that if there's some way in DevStats that I can find contributors from Turkey and then ask them to participate in our program as mentors.

Questions:
1. Is this possible technically? I see country maps and things like that, but the data is all aggregated.
2. Is this something that DevStats team would want to provide? My assumption is that DevStats uses public information and listing usernames and the countries shouldn't be an issue.

Pros:
- Easier to build local communities (at least for small countries or for countries that don't have too many contributors)
- Gamification (contributor X has N PRs, where as Y has M)

**Comment by aliok** [comment]
cc @jberkus @lukaszgryglicki @geekygirldawn @caniszczyk

**Comment by lukaszgryglicki** [comment, marked as answer]
I'm on PTO, but you can currently see contributors for a specific country: https://all.devstats.cncf.io/d/9/developer-activity-counts-by-repository-group-table?orgId=1&var-period_name=Last%20decade&var-metric=contributions&var-repogroup_name=All&var-country_name=Turkey
You can select metric type like contributions, PRs, issues, ..., date range like last month, last decade, .., CNCF project from repository group (or all): kubernetes, envoy, prometheus, ..., finally country (or all) like: Poland, Turkey, ...
You have github usernames there, I cannot show emails due to GDPR.
cc @caniszczyk

**Reply by aliok** [reply]
Wow, thanks! Exactly what I was looking for. I tried some other dashboards but not this one.

Thanks @lukaszgryglicki , you rock!

**Comment by lukaszgryglicki** [comment]
OK, so closing this.
